core/src/main/resources/jenkins/install/pluginSetupWizard_it.properties

```diff
@@ -1,51 +1,50 @@
 installWizard_welcomePanel_customizeActionTitle=Seleziona i plugin da installare
-installWizard_welcomePanel_customizeActionDetails=Seleziona e installa i plugin pi� adatti per le proprie necessit�.
-installWizard_jenkinsVersionTitle=Jenkins 
+installWizard_welcomePanel_customizeActionDetails=Seleziona e installa i plugin pi\u00F9 adatti per le proprie necessit\u00E0.
+installWizard_jenkinsVersionTitle=Jenkins
 installWizard_offline_title=Non in linea
-installWizard_offline_message=Sembra che quest''istanza di Jenkins non sia in linea. \
+installWizard_offline_message=Sembra che questa istanza di Jenkins non sia in linea. \
 <p style="font-size:18px; margin-top: 6%"> \
-Per informazioni sull''installazione di Jenkins in assenza di una connessione a Internet, si veda la \
+Per informazioni sull processo di Jenkins in assenza di una connessione a Internet, si veda la \
 <a href="https://jenkins.io/redirect/offline-installation" target="_blank">documentazione sull''installazione non in linea di Jenkins</a>. <br/><br/> \
-� possibile scegliere di continuare configurando un proxy o ignorando l''installazione dei plugin. \
-</p>
-installWizard_error_header=Si � verificato un errore
-installWizard_error_message=Si � verificato un errore durante l''installazione:
+\u00C8 possibile scegliere di continuare configurando un proxy o saltando la parte di installazione dei plugins.</p>
+installWizard_error_header=Si \u00E8 verificato un errore
+installWizard_error_message=Si \u00E8 verificato un errore durante il processo di installazione:
 installWizard_error_connection=Impossibile connettersi a Jenkins
-installWizard_error_restartNotSupported=Il riavvio non � supportato, riavviare manualmente quest''istanza
+installWizard_error_restartNotSupported=Il riavvio non \u00E8 supportato, riavviare manualmente questa istanza
 installWizard_installCustom_title=Per iniziare
 installWizard_installCustom_selectAll=Tutti
 installWizard_installCustom_selectNone=Nessuno
 installWizard_installCustom_selectRecommended=Suggeriti
 installWizard_installCustom_selected=Selezionati
 installWizard_installCustom_dependenciesPrefix=Dipendenze
-installWizard_installCustom_pluginListDesc=Si noti che qui non viene visualizzato l''elenco completo dei plugin. � possibile installare plugin aggiuntivi nel <strong>Gestore plugin</strong> una volta completata l''installazione iniziale. <a href="https://jenkins.io/redirect/installation-wizard" target="_blank">Si veda il Wiki per ulteriori informazioni</a>.
+installWizard_installCustom_pluginListDesc=Si noti che qui non viene visualizzata la lista completo di plugins. \u00C8 possibile installare plugin aggiuntivi nel <strong>Gestore plugin</strong> una volta completata la prima fase di installazione iniziale. <a href="https://jenkins.io/redirect/installation-wizard" target="_blank">Si veda il Wiki per ulteriori informazioni</a>.
```

Comment:
la lista completA di plugins

Reply:
also just change this "completata la prima fase di installazione iniziale" to "completata la fase di installazione iniziale"